x/wasm/internal/keeper/reflect_test.go

```diff
@@ -137,7 +137,7 @@ func TestMaskReflectContractSend(t *testing.T) {
 }
 
 func TestMaskReflectCustomMsg(t *testing.T) {
-	t.Skip("Alex: fails with `cannot protobuf JSON decode unsupported type: *types.Msg: failed to unmarshal JSON bytes`")
+	t.Skip("Alex: fails with `cannot protobuf JSON decode unsupported type: *types.Data: failed to unmarshal JSON bytes`")
```

Comment:
Yeah, this was a bit of an edge-case one. Leave it as skip and we address it before 2.0 with a different reflect contract likely.